blackbox/docs/admin/system-information.rst

```diff
@@ -849,6 +849,7 @@ then :ref:`gateway.recover_after_time <gateway.recover_after_time>` must not be
 set to ``0s``, otherwise the ``gateway.recover_after_nodes`` setting wouldn't
 have any effect.
 
+.. _sys_node_checks_disk_watermark_high:
```

Comment:
can we drop `_disk`?

Reply:
Why should we drop it?

Reply:
because these section IDs are used inline, and long IDs can mess up the visual presentation of hard wrapped paras. short (but intelligible) is ideal

Reply:
@mechanomi pushed a fixup